guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -26,6 +26,26 @@ include::topics/upgrading_satellite_server_parent.adoc[leveloffset=+2]
 // Upgrading a Connected Satellite Server
 include::topics/upgrading_a_connected_satellite_server.adoc[leveloffset=+3]
 
+ifdef::satellite[]
+// Migrating to a New Server
+include::common/modules/proc_migrating-to-a-new-server.adoc[leveloffset=+2]
+endif::[]
+
+ifdef::satellite[]
+// Creating_a_Backup_of_a_Server_on_rhel7_
+include::common/modules/proc_creating-a-backup-of-a-server-on-rhel7.adoc[leveloffset=+2]
+endif::[]
+
+ifdef::satellite[]
+// Performing_a_Fresh_Installation_of_a_Server_on_rhel8_
+include::common/modules/ref_performing-a-fresh-installation-of-a-server-on-rhel8.adoc[leveloffset=+2]
+endif::[]
+
+ifdef::satellite[]
+// Restoring a Restoring_a_Backup_of_a_Server_on_rhel8_
+include::common/modules/proc_restoring-a-backup-of-a-server-on-rhel8.adoc[leveloffset=+2]
+endif::[]
+
```

Comment:
If all files are only visible for Satellite, then you don't need individual "ifdef" + "endif" directives.

Can you please verify if the procedures also apply to Foreman and Katello?

Reply:
The current version of the doc is rather Satellite focused, but the goal is to have it for Foreman and Katello too

Reply:
I think I'd put all three into one big `ifdef::satellite …` because you will never enable individual steps of these, always all or nothing.